pkg/utils/utils.go

```diff
@@ -154,27 +153,27 @@ func GetComponentImage(component string) string {
 
 	envVarName, ok := COMPONENT_IMAGES[component]
 	if !ok {
-		logrus.Errorf("Environment variable name mapping missing for component: %s", component)
+		log.Info("Environment variable name mapping missing for component:", "component", component)
```

Comment:
this seems to be a more important message  then info

Reply:
this was an error before, why not keep it as `log.Error` ?

Reply:
fixed